SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -245,7 +245,7 @@ void ATPSCharacter::TakeGunDamageCrossServer_Implementation(float Damage, const
 
 This snippet implements the functionality that was previously contained within `TakeDamage` as a cross-server RPC called `TakeGunDamageCrossServer`.
 
-Because you’ve changed code in a function you now need build **ThirdPersonShooter.sln**, generate schema and a new snapshot. To do this:
+Because you have changed code in a function you now need to rebuild your project. Additionally, you need to generate schema and a new snapshot. To do this:
```

Comment:
```suggestion
Because you have changed code in a function, you now need to rebuild your project. Additionally, you need to generate schema and a new snapshot. To do this:
```
Quick question - since we've given a reason for generating the schema and snapshot above, should we give a reason here as well?

Reply:
@ernopp would you be able to comment here? Does the replication comment need to be repeated here also?

Reply:
"Additionally, because you've modified code related to replication, you need to generate schema and a new snapshot."

Reply:
Thanks both! Added changes !